File: POO_Semana1/Pytho_Semana1/Ejercicio03/Rectangle.java
class Producto{
    String Name= "";
    double Price = 0.0;
    int Amount = 0;

    double CalculateArea(){
        return Price*Amount;
    }
    double CalculatePerimeter(){
        return 2*(Price+Amount);
    }
}

public class Rectangle{
    public static void main(String[], args){

        Producto producto1 = new Producto();
        producto1.Name = "Leche";
        producto1.Price = 10.0;
        producto1.Amount = 4;
        double total_area = producto1.CalculateArea();
        double total_perimeter = producto1.CalculatePerimeter();

        System.out.println("El area es: " + total_area)
        System.out.println("El perimetro es: " + total_perimeter)

    }
}

File: POO_Semana1/Pytho_Semana1/Ejercicio03/Rectangle.py
class Rectangle:
    height = 0.0
    base = 0.0

    def CalculateArea(self):
        return self.height*self.base


    def CalculatePerimeter(self):
        return 2*(self.height+self.base)



rectangle1 = Rectangle()

rectangle1.height = 8

rectangle1.base = 4

rectangle2 = Rectangle()

rectangle2.height = 3

rectangle2.base = 2

total_area1 = rectangle1.CalculateArea()

total_perimeter1 = rectangle1.CalculatePerimeter()
print(f"El área es: {total_area1} , \n El perimetro es: {total_perimeter1}")

total_area2 = rectangle2.CalculateArea()

total_perimeter2 = rectangle2.CalculatePerimeter()

print(f"El área es: {total_area2} , \n El perimetro es: {total_perimeter2}")

